src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  margin: initial;
+  padding: initial;
```

Comment:
don't reset margins and paddings for *
use tags directly where you want to do it